storage/src/main/java/tech/pegasys/teku/storage/client/RecentChainData.java

```diff
@@ -377,6 +377,10 @@ public SpecVersion getCurrentSpec() {
     return getCurrentSlot().map(spec::atSlot).orElseGet(spec::getGenesisSpec);
   }
 
+  public Spec getSpec() {
```

Comment:
👍 I've been tempted to add this a few times but wasn't sure if it was cheating.  Think it's worth it to make it easier to get to the spec instead of having to pass it to so many different places alongside RecentChainData.

Reply:
Same here! I still kind of think its cheating / wonder if I should remove it, but `RecentChainData` already exposes some other spec-related API's, so I guess its fine 🤷‍♀️  Especially since we both keep running into this.